rpm/libsbc.spec

```diff
@@ -1,9 +1,8 @@
 Name:       libsbc
 
 Summary:    SBC library
-Version:    1.3
+Version:    1.4
 Release:    1
-Group:      System/Libraries
 License:    GPLv2
```

Comment:
Does this now match the license? Main package has COPYING.LIB which I assume is LGPLv2 and only tools has COPYING. (AFAIK you can have different licenses for subpackages.)

Reply:
Yes indeed. Updating license on this and the tools.